components.py

```diff
@@ -100,61 +102,102 @@ def define_property(class_name, property_name, property_definition, hubs_context
 
     if property_type == 'int':
         return IntProperty(
-            name=property_name
+            name=property_name,
+            description=property_definition.get("description") or "",
+            subtype=property_definition.get("subType") or "NONE",
```

Comment:
This looks interesting, what can we use subtypes on for these properties?

Reply:
There are a bunch of different sub types like ‘VELOCITY’,, ‘EULER’, ‘QUATERNION’, ‘AXISANGLE’, ‘XYZ’ that just show slightly nicer UI. There are also units like 'LENGTH", 'MASS', 'TIME', etc which will show the unit in the field. I don't know that these need to be part of any official component "schema' we come up with, but seems nice to expose as optional since it doesn't actually impact the data itself, just the presentation.